build-system/compile/post-closure-babel.js

```diff
@@ -40,10 +50,14 @@ async function terserMinify(code) {
       comments: /\/*/,
       // eslint-disable-next-line google-camelcase/google-camelcase
       keep_quoted_props: true,
-      preamble: ';',
     },
     sourceMap: true,
-  });
+  };
+  const basename = path.basename(filename, argv.esm ? '.mjs' : '.js');
+  if (mainBundles.includes(basename)) {
+    options.output.preamble = ';';
```

Comment:
Does `terser` omit the `preamble` if the value is null or undefined?

If so, this can be inlined back into the object literal...
```
{ ...
  preamble: mainBundles.includes(basename) ? ';' : undefined,
}
```

Reply:
applied. thanks